os/reading-the-system-log.md

```diff
@@ -63,5 +63,64 @@ journalctl -f
 journalctl -u apache.service -f
 ```
 
+## Red Entries With Line Wrapping
+
+Read logs using default `less` pager with line wrapping:
```

Comment:
Can we explain that we're just setting an env var before invoking the command?

And explain what the cryptic FRXMK value stands for?